src/base/_type.css

```diff
@@ -0,0 +1,127 @@
+/*
+
+  TYPE
+
+*/
+
+:root {
+  --font-weight-body: 200;
+  --font-weight-headings: 200;
+  --font-family-monospace: Menlo, Consolas, Monaco, monospace;
+}
+
+html {
+  line-height: 1.5; /* .lh-copy */
+  font-weight: var(--font-weight-body);
+}
+
+body {
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: var(--font-weight-headings);
+  line-height: 1.25; /* .lh-title */
+  margin: 0.75em 0;
+}
+
+h1 { font-size: 3rem;    } /* .f1 */
```

Comment:
Yeah, this would be preferable. Should we modify tachyons to use variables that we put in place to achieve consistency?

Reply:
Let's leave these for now. If we want to go forward with this method, I think we'd want to modify tachyons to use variables for every definition. This would potentially be a welcome PR for the entire Tachyons project, since it'd better allow folks to do what we're doing (which is what people need to do to pull it into a real core system for their CSS).

Reply:
Fine by me.